package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
what's the purpose of this new `package.json` file? why can't we have the dependencies in `backend/package.json` file?